Annotato/Annotato/Views/Document/Edit/DocumentEditViewController.swift

```diff
@@ -11,12 +13,25 @@ class DocumentEditViewController: UIViewController, AlertPresentable {
     }
 
     func initializeSubviews() {
+        initializeSpinner()
+
         initializeToolbar()
```

Comment:
```suggestion
        initializeSpinner()
        initializeToolbar()
        startSpinner()
```

Should we start here?

Reply:
I think this makes it hard to find the matching start and stop spinner actions, and can be prone to developer error in future?